src/main/kotlin/dev/stalla/model/itunes/PodcastItunes.kt

```diff
@@ -20,8 +20,8 @@ import dev.stalla.model.Person
  * @property complete The logical value of the `<itunes:complete>` field's text content.
  * @property type The `<itunes:type>` field text content.
  * @property owner The `<itunes:owner>` elements data as a [Person].
- * @property owner The `<itunes:title>` field text content.
- * @property owner The `<itunes:new-feed-url>` field text content.
+ * @property title The `<itunes:title>` field text content.
+ * @property newFeedUrl The `<itunes:new-feed-url>` field text content.
```

Comment:
Good that this detekt setup recognises these. I've been working on cleanup stuff in another branch last night were I also fixed this, but just because I've noticed them missing by coincidence.

Reply:
It was a good thing, I have enabled a bunch of comments rules I normally don't use (because I don't work on public libraries) and was impressed 👍